src/App.tsx

```diff
@@ -1,24 +1,168 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorModal } from './components/ErrorModal';
+import { LoadContext } from './Context/LoadContext';
 import { UserWarning } from './UserWarning';
+import { FilterType, ErrorType } from './typedefs';
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
+import { getFilteredTodos } from './helpers';
+import { useFilteredTodos } from './hooks/useFilteredTodos';
+import { useError } from './hooks/useError';
+import { usePost } from './hooks/usePost';
 
-const USER_ID = 0;
+const USER_ID = 6993;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.ALL);
+  const [loadingTodos, setLoadingTodos] = useState([0]);
+
+  const {
+    activeTodos,
+    completedTodos,
+  } = useFilteredTodos({ todos });
```

Comment:
As I think in this case u can pass just todos without an object)
```suggestion
  const {
    activeTodos,
    completedTodos,
  } = useFilteredTodos(todos);
```

Reply:
It doesn`t work ((